project/scaffold.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8-scaffold" % "0.13.0")
\ No newline at end of file
+addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8-scaffold" % "0.13.1")
```

Comment:
version 0.15.0 is available

Reply:
I tried to upgrade this further but it was incompatible with other dependencies. Can you get that version to work locally?

Reply:
No it doesnt work on mine either